### ISSUE TITLE
Video black screen in Android release mode - dev/3.0.0 branch

### ISSUE THREAD

**Issue by thangvq95** [AgoraIO/Flutter-SDK#165]

**Comment by LichKing-2234** [comment]
pls provide more information, such as SDK log and screen capture

**Comment by thangvq95** [comment]
Hello LichKing,

This in release mode so i can not log anything! it's working in debug mode and in ios (debug and release mode). Only doesn't work in android release mode (dev/3.0.0 branch)  I'm using flutter 1.17.5. I think this issue duplicate with #125 as well! 

Could you help me check this? Thanks @LichKing-2234 
![agora](https://user-images.githubusercontent.com/13799377/89260230-1240f580-d656-11ea-835d-1783e90e1ae1.jpg)

**Comment by LichKing-2234** [comment]
have you tried our example?

**Comment by LichKing-2234** [comment]
@thangvq95 Thanks for your issue, this problem cause by missing kotlin proguard setting.
```
-keepattributes *Annotation*
-keep class kotlin.** { *; }
-keep class org.jetbrains.** { *; }
```

**Comment by LichKing-2234** [comment]
fixed in 3.0.1-dev.5
